Classes/Asset/TagRenderer.php

```diff
@@ -135,6 +139,21 @@ private function getEntrypointLookup(string $buildName): EntrypointLookupInterfa
         return $this->entrypointLookupCollection->getEntrypointLookup($buildName);
     }
 
+    private function addAdditionalAbsRefPrefixDirectories(string $file)
+    {
+        $directories = GeneralUtility::trimExplode(
+            ',',
+            $GLOBALS['TYPO3_CONF_VARS']['FE']['additionalAbsRefPrefixDirectories'],
```

Comment:
Could you add a check if the additionalAbsRefPrefixDirectories already exists and is of type string